content/20.treatments.md

```diff
@@ -953,29 +953,3 @@ At present, a number of promising avenues continue to be explored.
 Potential therapeutics currently being studied target the SARS-CoV-2 or modify the host environment through many different mechanisms.
 Here, the relationship between the virus and several therapeutics described above are visualized.
 ](images/N001-LifeCyclePlusDrugs.png){#fig:therapeutics secno=1}
-
-##### Trained Immunity
```

Comment:
@agitter is this section proposed to be deleted? It used to be in a different part of the manuscript so I'm not sure if that's why it's showing up weird in the diff.

Reply:
Yes, those paragraphs were duplicated.  I left them in Section 5.2.4.3 and deleted them in Section 5.3.0.1.  I'm pretty sure the text is the same in both places but didn't do a diff.